<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qDMMLTransformSliders</class>
 <widget class="qDMMLWidget" name="qDMMLTransformSliders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transform</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="SlidersGroupBox">
     <property name="title">
      <string>Translation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="LRLabel">
        <property name="toolTip">
         <string>Left-Right</string>
        </property>
        <property name="text">
         <string>LR</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qDMMLLinearTransformSlider" name="LRSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>L&lt;--&gt;R</string>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="synchronizeSiblings">
         <set>ctkSliderWidget::NoSynchronize</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PALabel">
        <property name="toolTip">
         <string>Posterior-Anterior</string>
        </property>
        <property name="text">
         <string>PA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qDMMLLinearTransformSlider" name="PASlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>P&lt;--&gt;A</string>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="synchronizeSiblings">
         <set>ctkSliderWidget::NoSynchronize</set>
        </property>
        <property name="TypeOfTransform">
         <enum>qDMMLLinearTransformSlider::TRANSLATION_PA</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ISLabel">
        <property name="toolTip">
         <string>Inferior-Superior</string>
        </property>
        <property name="text">
         <string>IS</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="qDMMLLinearTransformSlider" name="ISSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>I&lt;--&gt;S</string>
        </property>
        <property name="minimum">
         <double>-200.000000000000000</double>
        </property>
        <property name="maximum">
         <double>200.000000000000000</double>
        </property>
        <property name="synchronizeSiblings">
         <set>ctkSliderWidget::NoSynchronize</set>
        </property>
        <property name="TypeOfTransform">
         <enum>qDMMLLinearTransformSlider::TRANSLATION_IS</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QWidget" name="MinMaxWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="MinLabel">
             <property name="text">
              <string>Min</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="qDMMLSpinBox" name="MinValueSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimalsOption">
              <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
             </property>
             <property name="minimum">
              <double>-1000000000.000000000000000</double>
             </property>
             <property name="value">
              <double>-200.000000000000000</double>
             </property>
             <property name="quantity">
              <string>length</string>
             </property>
             <property name="unitAwareProperties">
              <set>qDMMLSpinBox::Precision|qDMMLSpinBox::Prefix|qDMMLSpinBox::Scaling|qDMMLSpinBox::Suffix</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="MaxLabel">
             <property name="text">
              <string>Max</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="qDMMLSpinBox" name="MaxValueSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="decimalsOption">
              <set>ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts</set>
             </property>
             <property name="minimum">
              <double>-1000000000.000000000000000</double>
             </property>
             <property name="maximum">
              <double>1000000000.000000000000000</double>
             </property>
             <property name="value">
              <double>200.000000000000000</double>
             </property>
             <property name="quantity">
              <string>length</string>
             </property>
             <property name="unitAwareProperties">
              <set>qDMMLSpinBox::Precision|qDMMLSpinBox::Prefix|qDMMLSpinBox::Scaling|qDMMLSpinBox::Suffix</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qDMMLLinearTransformSlider</class>
   <extends>qDMMLSliderWidget</extends>
   <header>qDMMLLinearTransformSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLSliderWidget</class>
   <extends>ctkSliderWidget</extends>
   <header>qDMMLSliderWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qDMMLSpinBox</class>
   <extends>ctkDoubleSpinBox</extends>
   <header>qDMMLSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>qDMMLWidget</class>
   <extends>QWidget</extends>
   <header>qDMMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>ctkDoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MinValueSpinBox</sender>
   <signal>decimalsChanged(int)</signal>
   <receiver>MaxValueSpinBox</receiver>
   <slot>setDecimals(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MaxValueSpinBox</sender>
   <signal>decimalsChanged(int)</signal>
   <receiver>LRSlider</receiver>
   <slot>setDecimals(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LRSlider</sender>
   <signal>decimalsChanged(int)</signal>
   <receiver>MinValueSpinBox</receiver>
   <slot>setDecimals(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qDMMLTransformSliders</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>LRSlider</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qDMMLTransformSliders</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>PASlider</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qDMMLTransformSliders</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>ISSlider</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qDMMLTransformSliders</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>MaxValueSpinBox</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qDMMLTransformSliders</sender>
   <signal>dmmlSceneChanged(vtkDMMLScene*)</signal>
   <receiver>MinValueSpinBox</receiver>
   <slot>setDMMLScene(vtkDMMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
